### PR TITLE
Add performance handbook for EVM and Chainlink nodes

### PR DESCRIPTION
**Pull request by braddmiller**

### CODE REVIEW

**Comment by PatrickAlphaC** [comment]
@dwightjl we should have the link river team take a look at this. Can I pass it to them?

**Comment by PatrickAlphaC** [comment]
Shout out to @samsondav for the help making this.

**Comment by dwightjl** [comment]
> @dwightjl we should have the link river team take a look at this. Can I pass it to them?

Yes it would be great to get their review as well.

**Review by jmank88 — APPROVED** [review]

File: docs/Node Operators/evm-performance-configuration.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+If you are using a failover proxy transparently for commercial node provider services, it will continue to work properly as long as the RPC you are talking to acts just like a standard RPC node.
+
+You can have as many primary nodes as you want. Requests are evenly distributed across all nodes, so the performance increase will be linear as you add more nodes. If a node fails with no heads for several minutes or a failed liveness check, it is removed from the live pool and all requests are routed to one of the live nodes. If no live nodes are available, the system attempts to use nodes from the list of failed nodes at random.
+
+You can configure as many send-only nodes as you want. Send-only nodes only broadcast transactions and do not process regular RPC calls. Specifying additional send-only nodes uses a minimum number of RPC calls and can help to include transactions faster. Send-only nodes also act as backup if your primary node starts blackholing transactions.
```
Comment (samsondav) [inline docs/Node Operators/evm-performance-configuration.md:42]:
```suggestion
You can configure as many send-only nodes as you want. Send-only nodes only broadcast transactions and do not process regular RPC calls. Specifying additional send-only nodes uses a minimum number of RPC calls and can help to include transactions faster. Send-only nodes also act as backup if your primary node starts blackhole-ing transactions.
```

File: docs/Node Operators/evm-performance-configuration.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+- [Increasing transaction throughput](#increasing-transaction-throughput)
+  - [Increase ETH_MAX_QUEUED_TRANSACTIONS](#increase-eth_max_queued_transactions)
+  - [Increase ETH_MAX_IN_FLIGHT_TRANSACTIONS](#increasing-transaction-throughput)
+- [Optimizing RPC nodes](#optimizing-rpc-nodes)
+- [Remove rejections on expensive transactions](#remove-rejections-on-expensive-transactions)
+- [Increase ORM_MAX_OPEN_CONNS and ORM_MAX_IDLE_CONNS](#increase-orm_max_open_conns-and-orm_max_idle_conns)
+
+## Using multiple nodes
+
+Providing multiple primary nodes can improve performance and reliability, and is supported in Chainlink node version 1.3.0 and later. Chainlink nodes support configurations with multiple primary nodes and send-only nodes with automatic liveness detection and failover. It is no longer necessary to run a load balancing failover RPC proxy between Chainlink and its EVM RPC nodes.
```
Comment (samsondav) [inline docs/Node Operators/evm-performance-configuration.md:36]: Callouts are missing

**Review by samsondav — DISMISSED** [review]

**Review by dwightjl — APPROVED** [review]

**Comment by dwightjl** [comment]
Rebased to main

**Pull request merged**